### ISSUE TITLE
Http function?

### ISSUE THREAD

**Issue by pmortiz238** [TheRealAgentK/facebook-cf-sdk#26]
I am using CF   9,0,1,274733   and am getting the following error in function GetOAuthAccessToken:

"The method addParam is not defined"

```
public String function getOAuthAccessToken(String clientId = "", String clientSecret = "", String code = "", String grantType = "", String redirectUri) {
    var accessToken = "";
    var httpService = new Http(url="https://graph.facebook.com/oauth/access_token", timeout=variables.TIMEOUT);
    var queryString = "";
```

-- offending line below
##         if (arguments.clientId != "") httpService.addParam(type="url", name="client_id", value=arguments.clientId);

:
:
    }

Is "Http" a method that should exist in one of the cfc files?

thanks for any help...

**Comment by benorama** [comment]
Are you using the latest version of the SDK?
Anyway, this is really strange.

Can you use "Http" (cfhttp cfscript implementation) ?
http://help.adobe.com/en_US/ColdFusion/9.0/CFMLRef/WSe9cbe5cf462523a0693d5dae123bcd28f6d-7ff8.html

**Comment by pmortiz238** [comment]
Yep, v3.1.1.beta4...

I figured out that my installation of CF was not seeing the adobe http.cfc
file and created a custom tag folder pointing to the folder. The default
customtag folder was not available.

That problem was resolved. I get the login button to appear and now I get a
JS error: "Error: FB.Auth.setAuthResponse only compatible with OAuth2.
Source file: http://connect.facebook.net/en_US/all.js#xfbml=1
Line: 23
"

I think I saw a posting about that error and it should be fixed in the beta4
release. So I'm not sure why it's happening now...

Thanks...
Pablo
